### PR TITLE
[stable/coredns] Allow to specify a PodDisruptionBudget

### PR DESCRIPTION
**Pull request by mrueg**
#### Is this a new chart
No
#### What this PR does / why we need it:
Adds optional PodDisruptionBudget
#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

**Comment by mrueg** [comment]
@Smirl fyi since you were interested in that as well.

File: stable/coredns/templates/poddisruptionbudget.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+{{- end }}
+spec:
+  selector:
+    matchLabels:
+        app.kubernetes.io/instance: {{ .Release.Name | quote }}
+        {{- if .Values.isClusterService }}
+        k8s-app: {{ .Chart.Name | quote }}
+        {{- end }}
+        app.kubernetes.io/name: {{ template "coredns.name" . }}
+{{ toYaml .Values.podDisruptionBudget | indent 2 }}
```
Comment (Smirl) [inline stable/coredns/templates/poddisruptionbudget.yaml:27]: There are only 2 fields. I would perhaps add `minAvailable ` and `maxUnavailable` to values. Both way work though.

**Review by Smirl — APPROVED** [review]

File: stable/coredns/templates/poddisruptionbudget.yaml (hunk 1 above)
Comment (mrueg) [reply stable/coredns/templates/poddisruptionbudget.yaml:27]: I didn't include the fields to match with the other pdb in the Repo. I'd leave it like that

Comment (Smirl) [reply stable/coredns/templates/poddisruptionbudget.yaml:27]: Cool. Looks good to me. But I can't `/approve` 😿

**Comment by zanhsieh** [comment]
@Acaleph
@shashidharatd
@andor44
Little help with LGTM please.

File: stable/coredns/Chart.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: coredns
-version: 1.9.3
+version: 1.9.4
```
Comment (cpanato) [inline stable/coredns/Chart.yaml]:
```suggestion
version: 1.10.0
```
this is like a new feature and not bug fixing please bump to 1.10.0

**Review by cpanato — CHANGES_REQUESTED** [review]

**Comment by cpanato** [comment]
@mrueg please sing the DCO

**Comment by mrueg** [comment]
> @mrueg please sing the DCO

Yes shouldn't have used the github UI. :( Will repush.

**Comment by mrueg** [comment]
@cpanato pushed again with DCO signed.

**Review by cpanato — APPROVED** [review]
/lgtm

**Comment by mrueg** [comment]
Thanks, appreciated! @cpanato

**Pull request merged**